libraries/chain/include/steem/chain/smt_objects/smt_token_object.hpp

```diff
@@ -63,14 +69,17 @@ class smt_token_object : public object< smt_token_object_type, smt_token_object
    uint8_t              rel_amount_denom_bits = 0;
 };
 
+struct by_nai;
 struct by_control_account;
 
 typedef multi_index_container <
    smt_token_object,
    indexed_by <
       ordered_unique< tag< by_id >,
          member< smt_token_object, smt_token_id_type, &smt_token_object::id > >,
-      ordered_unique< tag< by_control_account >,
+      ordered_unique< tag< by_nai >,
```

Comment:
We could probably make this index by asset symbol type, but I don't think it is a blocker for this merge.

Reply:
Later PRs are going to modify this heavily.